### PR TITLE
fix(nvme): asynchronous io qpairs

### PR DESCRIPTION
**Pull request by mtzaurus**
Use asynchronous I/O qpair creation.

Signed-off-by: Mikhail Tcymbaliuk <mtzaurus@gmail.com>

### CODE REVIEW

**Comment by mtzaurus** [comment]
bors try

**Comment by mtzaurus** [comment]
bors try

**Comment by mtzaurus** [comment]
bors try

**Comment by mtzaurus** [comment]
bors try

**Review by dsavitskiy — APPROVED** [review]

**Comment by mtzaurus** [comment]
bors try

File: io-engine/src/bdev/nvmx/channel.rs [hunk 1]
```diff
@@ -596,12 +802,12 @@ impl NvmeControllerIoChannel {
             .build();
 
         // Connect qpair.
-        rc = qpair.connect();
-        if rc != 0 {
-            error!(?cname, ?rc, "failed to connect qpair");
-            poll_group.remove_qpair(&qpair);
-            return 1;
-        }
+        //rc = qpair.connect();
```
Comment (tiagolobocastro) [inline io-engine/src/bdev/nvmx/channel.rs]: not needed? then please remove

**Review by tiagolobocastro — APPROVED** [review]

**Comment by mtzaurus** [comment]
bors try

**Comment by mtzaurus** [comment]
bors try

**Comment by mtzaurus** [comment]
bors try

**Comment by mtzaurus** [comment]
bors try

**Comment by mtzaurus** [comment]
bors cancel

**Comment by mtzaurus** [comment]
bors try

**Comment by mtzaurus** [comment]
bors try

**Comment by mtzaurus** [comment]
bors try

**Comment by mtzaurus** [comment]
bors try

**Comment by mtzaurus** [comment]
bors try

**Comment by mtzaurus** [comment]
bors merge

**Pull request merged**